src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -167,6 +170,8 @@ public async Task GetInfoPackage()
             HttpContext.Response.Headers["Content-Disposition"] = contentDisposition;
             HttpContext.Response.Headers["Content-Type"] = "application/zip";
 
+            var debugInfoType = GetDebugInfoType();
+            var databases = GetStringValuesQueryString("databases", required: false);
```

Comment:
by convention, this should be `database` not `databases`

you are passing via url "database=X&database=Y" - passing 'databases' does not look natural here

Reply:
done